src/OrchardCore.Modules/OrchardCore.ContentFields/Drivers/ContentPickerFieldDisplayDriver.cs

```diff
@@ -0,0 +1,107 @@
+        {
+            return Initialize<DisplayContentPickerFieldViewModel>("ContentPickerField", async model =>
+            {
+                model.Field = field;
+                model.Part = context.ContentPart;
+                model.PartFieldDefinition = context.PartFieldDefinition;
+                model.Updater = context.Updater;
+                model.SelectedContentItemIds = string.Join(",", field.ContentItemIds);
+                model.ContentItems = await _session.Query<ContentItem, ContentItemIndex>()
+                    .With<ContentItemIndex>(x => x.ContentItemId.IsIn(field.ContentItemIds) && x.Published)
+                    .ListAsync();
+            })
+            .Location("Content")
+            .Location("SummaryAdmin", "");
+        }
+
+        public override IDisplayResult Edit(ContentPickerField field, BuildFieldEditorContext context)
+        {
+            return Initialize<EditContentPickerFieldViewModel>(GetEditorShapeType(context), async model =>
+            {
+                model.ContentItemIds = field.ContentItemIds == null ? string.Empty : string.Join(",", field.ContentItemIds);
+
+                model.Field = field;
+                model.Part = context.ContentPart;
+                model.PartFieldDefinition = context.PartFieldDefinition;
+
+                model.SelectedItems = new List<ContentPickerItemViewModel>();
+
+                foreach (var contentItemId in field.ContentItemIds ?? new string[0])
+                {
+                    var contentItem = await _contentManager.GetAsync(contentItemId);
+                    var contentItemMetadata = await _contentManager.PopulateAspectAsync<ContentItemMetadata>(contentItem);
+                    model.SelectedItems.Add(new ContentPickerItemViewModel
+                    {
+                        ContentItemId = contentItemId,
+                        DisplayText = contentItemMetadata.DisplayText
+                    });
+                }
+            });
+        }
+
+        public override async Task<IDisplayResult> UpdateAsync(ContentPickerField field, IUpdateModel updater, UpdateFieldEditorContext context)
+        {
+            var viewModel = new EditContentPickerFieldViewModel();
+
+            var modelUpdated = await updater.TryUpdateModelAsync(viewModel, Prefix, f => f.ContentItemIds);
+
+            if (!modelUpdated)
+            {
+                return Edit(field, context);
+            }
+
+            field.ContentItemIds = viewModel.ContentItemIds == null
+                ? new string[0] : viewModel.ContentItemIds.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
+
+            var settings = context.PartFieldDefinition.Settings.ToObject<ContentPickerFieldSettings>();
+
+            if (settings.Required && field.ContentItemIds.Length == 0)
+            {
+                updater.ModelState.AddModelError(Prefix, T[$"The {context.PartFieldDefinition.DisplayName()} field is required."]);
```

Comment:
`T["The {0} field is required.", context.PartFieldDefinition.DisplayName()]`

Otherwise the string to translate is dynamic, and there will be no way to translate it.